Functions_Lab.js

```diff
@@ -0,0 +1,339 @@
+/* Question 1
```

Comment:
Good demonstration of how to use functions to solve problems. I think you have good understanding of logics and syntax of how to write an effective function. Good job and keep it up!

Reply:
Thanks a lot JR!
I will do my best